app/managers.py

```diff
@@ -0,0 +1,42 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self):
+        self._connection = sqlite3.connect("cinema.sqlite")
+        self.table_name = "actors"
+
+    def create(self, first_name: str, last_name: str):
+        query = f"""
+        INSERT INTO {self.table_name} (first_name, last_name)
+        VALUES (?, ?);
+        """
+        self._connection.execute(query, (first_name, last_name))
+        self._connection.commit()
+
+    def all(self):
+        literary_format_cursor = self._connection.execute(

```

Comment:
The variable name `literary_format_cursor` is not very descriptive of its purpose. Consider renaming it to something more intuitive like `cursor` or `result_cursor` to better convey its role in fetching data from the database.